hr_employee_relative/views/hr_employee.xml

```diff
@@ -9,9 +9,18 @@
         <field name="model">hr.employee</field>
         <field name="inherit_id" ref="hr.view_employee_form" />
         <field name="arch" type="xml">
-            <field name="spouse_complete_name" position="replace" />
-            <field name="spouse_birthdate" position="replace" />
-            <field name="children" position="replace" />
+            <field name="spouse_complete_name" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </field>
+            <field name="spouse_birthdate" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </field>
+            <field name="children" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </field>
+            <xpath expr="//field[@name='children']/.." position="attributes">
```

Comment:
If you hide the group it is necessary to hide all the fields to?

Reply:
You are right! Thank you :smile: